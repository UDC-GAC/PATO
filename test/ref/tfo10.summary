# Sequence-ID	TFOs (abs)	TFOs (rel)	GA (abs)	GA (rel)	TC (abs)	TC (rel)	GT (abs)	GT (rel)
ENSLCOT00000000003.1	259	0.0108	0	0	72	0.00302	187	0.00783	
ENSLCOT00000000010.1	319	0.0137	0	0	49	0.0021	270	0.0116	
ENSLCOT00000000046.1	23	0.00096	3	0.000125	19	0.000793	1	4.17e-05	
ENSLCOT00000000102.1	95	0.00362	0	0	1	3.81e-05	94	0.00359	
ENSLCOT00000000118.1	14	0.00115	4	0.000329	10	0.000823	0	0	
ENSLCOT00000000120.1	114	0.00506	1	4.44e-05	0	0	113	0.00501	
ENSLCOT00000000128.1	5	0.000175	5	0.000175	0	0	0	0	
ENSLCOT00000000142.1	23	0.000823	0	0	16	0.000573	7	0.00025	
ENSLCOT00000000188.1	25	0.00103	18	0.000738	0	0	7	0.000287	
ENSLCOT00000000235.1	194	0.0219	0	0	194	0.0219	0	0	
ENSLCOT00000000240.1	33	0.00104	32	0.00101	1	3.15e-05	0	0	
ENSLCOT00000000242.1	200	0.0136	200	0.0136	0	0	0	0	
ENSLCOT00000000271.1	26	0.000843	16	0.000519	10	0.000324	0	0	
ENSLCOT00000000350.1	86	0.00421	7	0.000342	78	0.00381	1	4.89e-05	
ENSLCOT00000000543.1	1	0.000118	1	0.000118	0	0	0	0	
ENSLCOT00000000576.1	4	0.000258	4	0.000258	0	0	0	0	
ENSLCOT00000000577.1	11	0.000286	5	0.00013	4	0.000104	2	5.21e-05	
ENSLCOT00000000578.1	11	0.000299	5	0.000136	4	0.000109	2	5.43e-05	
ENSLCOT00000000580.1	6	0.000185	1	3.09e-05	4	0.000124	1	3.09e-05	
ENSLCOT00000000582.1	103	0.00273	98	0.0026	4	0.000106	1	2.65e-05	
ENSLCOT00000000621.1	6	0.000217	6	0.000217	0	0	0	0	
ENSLCOT00000000730.1	42	0.00175	0	0	42	0.00175	0	0	
ENSLCOT00000000954.1	16	0.000333	8	0.000167	8	0.000167	0	0	
ENSLCOT00000000955.1	26	0.000474	6	0.000109	20	0.000364	0	0	
ENSLCOT00000000983.1	3	0.000436	3	0.000436	0	0	0	0	
ENSLCOT00000000988.1	4	0.000131	3	9.81e-05	0	0	1	3.27e-05	
ENSLCOT00000000989.1	148	0.00827	15	0.000838	7	0.000391	126	0.00704	
ENSLCOT00000000990.1	7	0.000424	0	0	7	0.000424	0	0	
ENSLCOT00000000991.1	7	0.000521	0	0	7	0.000521	0	0	
ENSLCOT00000000992.1	7	0.000649	0	0	7	0.000649	0	0	
ENSLCOT00000000993.1	7	0.000485	0	0	7	0.000485	0	0	
ENSLCOT00000001086.1	62	0.00314	3	0.000152	58	0.00294	1	5.06e-05	
ENSLCOT00000001108.1	177	0.00512	47	0.00136	17	0.000492	113	0.00327	
ENSLCOT00000001157.1	3	0.000324	1	0.000108	1	0.000108	1	0.000108	
ENSLCOT00000001262.1	10	0.00038	10	0.00038	0	0	0	0	
ENSLCOT00000001311.1	10	0.00169	0	0	0	0	10	0.00169	
ENSLCOT00000001312.1	3	0.000557	0	0	3	0.000557	0	0	
ENSLCOT00000001324.1	148	0.0121	41	0.00334	3	0.000244	104	0.00847	
ENSLCOT00000001405.1	485	0.0151	475	0.0148	10	0.000312	0	0	
ENSLCOT00000001437.1	42	0.00123	37	0.00109	5	0.000147	0	0	
