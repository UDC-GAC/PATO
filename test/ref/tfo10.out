# Sequence-ID	Start	End	Score	Motif	Error-rate	Errors	Guanine-rate	Duplicates	TFO	Duplicate locations
ENSLCOT00000000003.1	77	96	16	Y	0.16		0.79	-1	TCTCCCCCCTCTCCCCCCC	-
ENSLCOT00000000003.1	712	733	19	Y	0.095	o4	0.81	-1	CCCTGCCCCTCCCCCCCTCCC	-
ENSLCOT00000000003.1	29	45	16	M	0		0.5	-1	GGGTGTTTTTTGGGGT	-
ENSLCOT00000000003.1	128	144	16	M	0		0.56	-1	GGGTGGTTTTTGGGGT	-
ENSLCOT00000000003.1	168	184	16	M	0		0.62	-1	GGGTGGGTTTTGGGGT	-
ENSLCOT00000000003.1	208	224	16	M	0		0.56	-1	GGGTGGTTTTTGGGGT	-
ENSLCOT00000000003.1	228	244	16	M	0		0.5	-1	GGGTGTTTTTTGGGGT	-
ENSLCOT00000000003.1	288	304	16	M	0		0.5	-1	GGGTGTTTTTTGGGGT	-
ENSLCOT00000000003.1	388	404	16	M	0		0.56	-1	GGGTGGTTTTTGGGGT	-
ENSLCOT00000000003.1	408	424	16	M	0		0.56	-1	GGGTGGTTTTTGGGGT	-
ENSLCOT00000000003.1	467	483	16	M	0		0.56	-1	GGGTGGTTTTTGGGGT	-
ENSLCOT00000000003.1	546	562	16	M	0		0.56	-1	GGGTGGTTTTTGGGGT	-
ENSLCOT00000000003.1	605	622	16	M	0.059		0.53	-1	GGGTGGTTTTTTGGGGT	-
ENSLCOT00000000010.1	260	285	19	Y	0.24	o6	0.68	-1	CCCCCCGCCCCTTTCTCCCTCCTTC	-
ENSLCOT00000000010.1	36	52	16	M	0		0.69	-1	GGTTGTGGGTGGGGGT	-
ENSLCOT00000000010.1	54	82	16	M	0.43	o25	0.57	-1	TTTGTGGGTGGGGTTGTGGGTGTGGCTG	-
ENSLCOT00000000010.1	174	202	16	M	0.43	o26	0.61	-1	GTGGGTGGGGTTGTTTGTGGGTGGGGCT	-
ENSLCOT00000000102.1	651	668	16	M	0.059		0.53	-1	GTGTTGGGTTTTTGGGG	-
ENSLCOT00000000118.1	332	355	19	Y	0.17	o11	0.22	-1	TTTTTCCTTTTGCTTTCCTTTTT	-
ENSLCOT00000000120.1	731	758	19	M	0.3	o1	0.67	-1	TCTGGGGGGGTGTGTGGGGGGGTTGTG	-
ENSLCOT00000000128.1	47	67	19	R	0.05	o11	0.25	-1	GAAAGAAGAAGCAGAAAAAA	-
ENSLCOT00000000142.1	965	981	16	Y	0		0.75	-1	CTCCCCTCCCTCCCCT	-
ENSLCOT00000000235.1	37	81	19	Y	0.57	o8o23	0.48	-1	CTTTTCCCATCTTTCCCCCTCCCATTCCTCCTTTCTTTTTTCCC	-
ENSLCOT00000000240.1	338	361	19	R	0.17	o1o20	0.35	-1	ATGAGGAGAGGAAAGAAAAATGA	-
ENSLCOT00000000242.1	500	527	16	R	0.41		0.48	-1	GAAAAAGGGGGAAAAAAAGAGAGGGGG	-
ENSLCOT00000000242.1	562	580	16	R	0.11		0.28	-1	AAGAGAGAAAAAAGAAAG	-
ENSLCOT00000000242.1	588	610	19	R	0.14	o18	0.5	-1	GAGAGGAGAGAAGAGAGGCAGA	-
ENSLCOT00000000271.1	778	799	19	R	0.095	o14	0.57	-1	AGGAGGAAGAGAAGCAGGGGG	-
ENSLCOT00000000350.1	554	574	19	M	0.05	o9	0.7	-1	TGGTGGGGTCTGGGGGTGGG	-
ENSLCOT00000000543.1	352	372	19	R	0.05	o11	0.4	-1	GAAGAAGAAGATGAAGAAGG	-
ENSLCOT00000000577.1	1122	1142	19	M	0.05	o9	0.4	-1	TTGTTGGTTAGTTTGGTGTG	-
ENSLCOT00000000578.1	1053	1073	19	M	0.05	o9	0.4	-1	TTGTTGGTTAGTTTGGTGTG	-
ENSLCOT00000000580.1	859	879	19	M	0.05	o9	0.4	-1	TTGTTGGTTAGTTTGGTGTG	-
ENSLCOT00000000582.1	197	230	19	R	0.42	o9o29	0.12	-1	AAGAAAAAATAAGAAAGAAAAAAAGAAAATAAA	-
ENSLCOT00000000582.1	1092	1112	19	M	0.05	o9	0.4	-1	TTGTTGGTTAGTTTGGTGTG	-
ENSLCOT00000000730.1	990	1017	19	Y	0.3	o13	0.41	-1	TCCTTTCTTCTCCATCCTTTCTTCTTC	-
ENSLCOT00000000954.1	1885	1905	19	Y	0.05	o10	0.4	-1	CTCCTTCCCTGTTTTTTTCC	-
ENSLCOT00000000989.1	269	289	19	Y	0.05	o14	0.35	-1	TCTCTTTTTCTCTCACTTCT	-
ENSLCOT00000000989.1	446	462	16	R	0		0.44	-1	GAGAAGAGAAGGGAAA	-
ENSLCOT00000000989.1	641	657	16	M	0		0.38	-1	TGGTTTTGGTTGTTTG	-
ENSLCOT00000000990.1	269	289	19	Y	0.05	o14	0.35	-1	TCTCTTTTTCTCTCACTTCT	-
ENSLCOT00000000991.1	269	289	19	Y	0.05	o14	0.35	-1	TCTCTTTTTCTCTCACTTCT	-
ENSLCOT00000000992.1	269	289	19	Y	0.05	o14	0.35	-1	TCTCTTTTTCTCTCACTTCT	-
ENSLCOT00000000993.1	269	289	19	Y	0.05	o14	0.35	-1	TCTCTTTTTCTCTCACTTCT	-
ENSLCOT00000001086.1	715	742	19	Y	0.3	o5o25	0.33	-1	CTCTTATCCTTCCTTTCTTTTCCTTAT	-
ENSLCOT00000001108.1	938	965	19	Y	0.3	o5o21	0.52	-1	CTTCCATTCTCCTCTCCCTTCATCTCC	-
ENSLCOT00000001108.1	38	57	16	R	0.16		0.53	-1	GGGAAAGGGAGGGGAAAAA	-
ENSLCOT00000001108.1	1025	1045	19	R	0.05	o4	0.6	-1	AGGGTGGGGAGGAGAAGAAG	-
ENSLCOT00000001108.1	277	333	19	M	0.66	o4o24o34o46	0.32	-1	GTTGAGGTTTGTGTTTGTTTGGTTCGGTTTGGGGCGGTTTTTTTGTCTTTTTTTTT	-
ENSLCOT00000001324.1	175	194	16	R	0.16		0.42	-1	AAGGGGAAGAAAGGAAAAG	-
ENSLCOT00000001324.1	423	452	19	M	0.34	o2o22	0.52	-1	TTCGTTGTTGTGTTTTGGGGGGAGGGTGG	-
ENSLCOT00000001405.1	366	409	19	R	0.56	o31	0.4	-1	GGAAAAAAGGAGAGGAGGAAGAGGAAGAAAATGAAGAAAGAGA	-
ENSLCOT00000001405.1	536	564	19	R	0.32	o2o25	0.46	-1	AGTGAGAGGAAGAGGAGAAGAGGAACAG	-
ENSLCOT00000001437.1	1084	1105	19	Y	0.095	o8	0.52	-1	CCCTTCTTGCTCCTCTCCCTT	-
ENSLCOT00000001437.1	717	742	19	R	0.24	o11	0.36	-1	AAGAAAGAAGACAAAGAGGAGAAGG	-
